src/axom/spin/MortonIndex.hpp

```diff
@@ -148,6 +148,7 @@ struct MortonBase
    * so, e.g. in 2D, 6 == 0b0110 becomes 0b*0*1*1*0 == 0b00010100 == 20
    */
   AXOM_HOST_DEVICE
+  AXOM_SUPPRESS_UBSAN
```

Comment:
I've seen warnings on this line in msvc. 
What is the UBSAN error/warning for this?

I think it's related to compile time constants and we might be able to fix it by casting to the appropriate type.

Reply:
> What is the UBSAN error/warning for this?

The errors are of the form:

`../../axom/spin/MortonIndex.hpp:156: runtime error: left shift of 50331812 <or another large value> by 8 places cannot be represented in type 'int'`